docs/public/specifications/test-zones/Zone-TP/zone11.md

```diff
@@ -0,0 +1,340 @@
+
+* Zone: "." (root zone)
+  * It is served by ns-non-null-spf.root-servers.zone11.xa.
+  * The TXT RRset at the apex of the root zone contains one TXT record with an
+    SPF record that isn’t null (i.e. permits at least one IP address).
+
+### NON-NULL-SPF-TLD-ZONE
+
+A TLD zone with a non-null SPF record.
+
+* Zone: non-null-spf-zone11
+  * It is served by ns1 and ns2.zone11.xa.
+  * The TXT RRset at the apex of the TLD zone contains one TXT record with an
+    SPF record that isn’t null (i.e. permits at least one IP address).
+
+### NON-NULL-SPF-ARPA-ZONE
+
+A zone in .arpa with a non-null SPF record.
+
+* Zone: non-null-spf-arpa-zone.zone11.arpa
+  * It is served by ns-non-null-spf.root-servers.zone11.xa.
+  * The TXT RRset at the apex of the zone contains one TXT record with an SPF
+    record that isn’t null (i.e. permits at least one IP address).
+
+### INVALID-SYNTAX-1
+
+A zone whose SPF record is syntactically invalid.
+
+* Zone: invalid-syntax-1.zone11.xa
+  * The TXT RRset at the apex contains one SPF record with invalid syntax,
+    i.e. beginning with `v=spf1` followed by whitespace but not compliant with
+    the ABNF grammar described in [RFC 7208, Section 4.5][RFC 7208#4.5] and
+    following sections.
+
+### INVALID-SYNTAX-2
+
+Like INVALID-SYNTAX-1, but two of the name server names in the delegation and in
+the zone point to the same IP address. This scenario is there to ensure that
+the all name servers that serve an ill-formed SPF policy are reported, even if
+two of them have the same IP address.
+
+* Zone: invalid-syntax-2.zone11.xa
+  * It is served by ns1a, ns1b, and ns2.invalid-syntax.zone11.xa.
+  * ns1a and ns1b resolve to the same IPv4 and IPv6 address as
+    ns1.child.zone11.xa.
+  * ns2 resolves to the same IPv4 and IPv6 address as ns2.child.zone11.xa.
+  * The TXT RRset at the apex contains one SPF record with invalid syntax,
+    i.e. beginning with `v=spf1` followed by whitespace but not compliant with
+    the ABNF grammar described in [RFC 7208, Section 4.5][RFC 7208#4.5] and
+    following sections.
+
+### INVALID-SYNTAX-3
+
+A zone whose SPF record contains a few random bytes, with a few non-ASCII
+bytes among those.
+
+* Zone: invalid-syntax-3.zone11.xa
+  * The TXT RRset at the apex contains one SPF record beginning with `v=spf1`,
+    followed by some whitespace and containing some random bytes outside the
+    printable ASCII range, which is not compliant with the ABNF grammar
+    described in [RFC 7208, Section 4.5][RFC 7208#4.5] and following sections.
+
+### NON-AUTH-TXT
+
+A zone containing one SPF record in its TXT RRset, but served from name
+servers that fail to set the AA bit in their responses.
+
+* Zone: non-auth-txt.zone11.xa
+  * The TXT RRset at the apex contains one SPF record
+  * Both name servers serve the aforementioned TXT RRset with the AA bit unset
+    in the response header.
+
+### SERVFAIL
+
+A zone where queries for the TXT RRset are responded to with an RCODE

```

Comment:
```suggestion
A zone where queries for the TXT RRset are responded with an RCODE
```

Reply:
It’s actually grammatically correct, but I’ll rephrase anyway.